feature_engine/transformation/log.py

```diff
@@ -147,6 +149,45 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X.loc[:, self.variables_] = np.log10(X.loc[:, self.variables_])
 
         return X
+    
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Transforms the variables using exp transformation.
+
+        Parameters
+        ----------
+        X: Pandas DataFrame of shape = [n_samples, n_features]
+            The data to be transformed.
+
+        Raises
+        ------
+        TypeError
+            If the input is not a Pandas DataFrame
+        ValueError
+            - If the variable(s) contain null values
+            - If the df has different number of features than the df used in fit()
+            - If some variables contains zero or negative values
```

Comment:
contain instead of contain**s**

Reply:
Already says "contain". 
Are you asking to add the "S"? What about line 127 in transform method?